### ISSUE TITLE
Matt 10:28  φοβεῖσθε

### ISSUE THREAD

**Issue by jcuenod** [biblicalhumanities/Nestle1904#14]
Is this an error on line 5337 in `Nestle1904.csv`?

```
Matt 10:28  φοβεῖσθε    V-PEM-2P@@V-PNM-2P  V-PEM-2P@@V-PNM-2P  5399    φοβέω   φοβεῖσθε    V-PEM-2P    V-PNM-2P
```

I'm seeing some weird `@@` symbols (which only occur on this line) and a couple of extra columns at the end...

**Comment by emg** [comment]
Yes, this is an error. It is a leftover from the process of preparing the text – the @@ separates two alternative analyses that needed to be disambiguated. They were overlooked and did not get disambiguated. I need to dig in and see how to resolve this – it is only slightly complicated, but enough that I can't say for sure how to resolve it just by looking at this line.
